packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1082,9 +1082,11 @@ export class Profiles extends ProfilesCache {
                 _: [""],
             };
             for (const prop of Object.keys(profSchema)) {
-                cmdArgs[prop] = serviceProfile.profile[prop] ? serviceProfile.profile[prop] : baseProfile.profile[prop];
+                cmdArgs[prop] = serviceProfile.profile[prop]
+                    ? serviceProfile.profile[prop]
+                    : baseProfile?.profile[prop];
             }
-            if (baseProfile) {
+            if (baseProfile || (serviceProfile.profile.tokenType && serviceProfile.profile.tokenValue)) {
```

Comment:
@lauren-li I don't believe both, tokenType and tokenValue, are needed for the check.  Extender profiles that support this will have both or none depending on the authentication method chosen

Reply:
Thanks for looking at this, @JillieBeanSim! Per our discussion, I will close this PR in favor of adding the initial check for base profiles before running `getCombinedProfiles()`.